relentless/ensemble.py

```diff
@@ -125,15 +172,32 @@ def reset(self):
                 self._N[t] = None
             if 'mu_{}'.format(t) in self.conjugates:
                 self._mu[t] = None
-        self.rdf = core.PairMatrix(self.types)
+        self.rdf = core.PairMatrix(types=self.types)
 
         return self
 
     def copy(self):
+        """Copies the ensemble and returns the copy with reset parameters.
+
+        Returns
+        -------
+        :py:class:`Ensemble`
+            A copy of the ensemble object with reset parameters.
+
+        """
         ens = copy.deepcopy(self)
         return ens.reset()
```

Comment:
I don't know why I wanted to reset this ensemble on copy, but it does not make sense to do that. The user can always do:
```py
ens2 = ens.copy().clear()
```
Let's remove the call to `reset` here.